hummingbird/ml/operator_converters/sklearn/sv.py

```diff
@@ -21,6 +21,7 @@ def __init__(self, kernel, degree, sv, nv, a, b, gamma, coef0, classes, device):
         super(SVC, self).__init__(classification=True)
         self.kernel = kernel
         self.degree = degree
+        # self.gamma = torch.nn.Parameter(torch.from_numpy(np.array(gamma, dtype=np.float64)), requires_grad=False)
```

Comment:
why the commented-out line? should this be removed?

Reply:
Ops, I think this is just some leftover from some test I was doing. I remove those.